apps/passport-server/.env.local.example

```diff
@@ -61,9 +61,9 @@ PASSPORT_CLIENT_URL="http://localhost:3000"
 # using the following commands at the root of the project, after installing dependencies and building
```

Comment:
should we add `IS_LOCAL_SERVER=true` to here?

Reply:
It's at the bottom! But set to `false` by default, because it requires some extra steps to make it work.